src/res/flame.html

```diff
@@ -258,10 +318,28 @@ <h1>/*title:*/</h1>
 		search(true);
 	}
 
+	document.getElementById('legend').onclick = function() {
+		legend = !legend;
```

Comment:
This variable ^^^ isn’t reset to false when the popup closes, so the user must click the "legend" button twice to reopen it. Since the pop-up overlays the page, it’s probably ok to remove this flag altogether.